tests/integration/hosts/maya/input/startup/userSetup.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import sys
+
+from maya import cmds
+
+import pyblish.util
+
+
+def setup_pyblish_logging():
+    log = logging.getLogger("pyblish")
+    hnd = logging.StreamHandler(sys.stdout)
+    fmt = logging.Formatter(
+        "pyblish (%(levelname)s) (line: %(lineno)d) %(name)s:"
+        "\n%(message)s"
+    )
+    hnd.setFormatter(fmt)
+    log.addHandler(hnd)
+
+
+def _run_publish_test_deferred():
+    try:
+        pyblish.util.publish()
+    finally:
+        cmds.quit(force=True)
+
+
+cmds.evalDeferred("setup_pyblish_logging()", evaluateNext=True)
+cmds.evalDeferred("_run_publish_test_deferred()", lowestPriority=True)
```

Comment:
Still I feel like you're best off registering a single callback instead of multiple. You are much better off just doing:
```python
cmds.evalDeferred("_run_publish_test_deferred()", lowestPriority=True)
```
And having the call to setup publish logging be at the start of `_run_publish_test_deferred`. Or what's the reason you're still separating the too?